bundler/spec/fixtures/ruby/gemspecs/require_ruby_3

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
```

Comment:
```suggestion
# frozen_string_literal: true

```

Should there be a newline here? `grep -A 1 -r frozen_string .` says probably...
Meh, since when am I rubocop.